RELEASING.md

```diff
@@ -12,4 +12,79 @@ Create a git tag and a new release on GitHub.
 
 # Publishing the new App
 
-## TODO
\ No newline at end of file
+In case this is a Hathor release, make sure you also read our internal guide in https://github.com/HathorNetwork/ops-tools/blob/master/docs/release-guides/hathor-wallet.md
+
+1. Run the `release.sh` script, which will clean the environment and build the app for all platforms. The files go to the `dist` folder after the script finishes running. You should get 4 of them: `.AppImage`, `.deb`, `.dmg` and `.exe`.
+
+2. Sign all 4 files with the following command:
+
+```
+for file in *.AppImage *.deb *.dmg *.exe; do gpg --output "$file.sig" --detach-sig "$file"; done
+```
+
+This will generate four signatures with the `.sig` extension.
+
+You should upload the 4 files and the 4 signatures to the GitHub release.
+
+# Signature verification
+
+To verify the signature of one of our packages, you can download it and its corresponding signature and use the following command:
+
+```
+gpg --verify <file>.sig <file>
+```
+
+For this to work, you must have the public key of the signer in your keyring. Check [Our public keys](#our-public-keys) to see how to add them. Otherwise, you'll get errors like this when trying to verify:
+
+```
+gpg: Can't check signature: No public key
+```
+
+If you did add our public keys, you may still get an error like this:
+
+TODO: Change the name in this example.
+
+```
+gpg: Good signature from "Werner Koch (dist sig)" [unknown]
```

Comment:
I feel it's better to use an actual builder as an example in this doc.

Reply:
This was just a placeholder while I didn't get the actual builders.